v3/integration/lints/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/zmap/zlint/v3/integration/lints
```

Comment:
I haven't worked on a Go project that uses submodules yet. Is there a reason that you wanted to use one here? Could we get away with keeping this tool in the main module?

Reply:
I think most of this code should be behind the integration build tag as well.

Reply:
> Is there a reason that you wanted to use one here?

So the "zen" of Go modules that I have heard before is that they are a collection of packages that are meant to be versioned together. So I suppose in this case it would be because that, while this little code linter is used to lint ZLint, it is not really versioned with ZLint itself.

...at least that is the "grasping at straws" version. Really, it's because I tend to give each `main` it's own module because I see them all as standalone projects that just so happen to be close to each other.

Reply:
> I think most of this code should be behind the integration build tag as well 

I admit I'm not sure what you're referring to here. Is there some bundling script I should be addressing to make sure this doesn't get packaged into the shippable packages?

Reply:
> ...at least that is the "grasping at straws" version. Really, it's because I tend to give each main it's own module because I see them all as standalone projects that just so happen to be close to each other.

If you don't feel super strongly I'd lightly favour dropping the submodule to stick with what we know. It seems extremely unlikely that we'll version the lint separately.

> I admit I'm not sure what you're referring to here. Is there some bundling script I should be addressing to make sure this doesn't get packaged into the shippable packages?

Looking closer I _think_ we're ok here because the `.goreleaser.yml` config only includes the `zlint` binary: https://github.com/zmap/zlint/blob/master/v3/.goreleaser.yml#L8

We already have the `zlint-gtld-update` command and it isn't present in our binary releases so I think I've overthinking it here and you're OK without a build tag.